my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+export const CustomInput = props => {
+  const { label, input, meta } = props;
+  return (
+    <Fragment>
+      <label> {props.label} </label>
```

Comment:
Don´t leave spaces between the tags and the content.
`<label> {props.label} </label>`
`<label>{props.label}</label>`